website/blog/2021-07-29-Introduction to Flint.md

```diff
@@ -0,0 +1,92 @@
+---
+slug: intro to flint
```

Comment:
This is a wrong slug. Read about how slugs are formatted [here](https://docusaurus.io/docs/docs-introduction)